src/BaGet/Program.cs

```diff
@@ -51,6 +53,17 @@ await provider
                     // Remove the upload limit from Kestrel. If needed, an upload limit can
                     // be enforced by a reverse proxy server, like IIS.
                     options.Limits.MaxRequestBodySize = null;
+                })
+                .ConfigureAppConfiguration((builderContext, config) =>
+                {
+                    var root = Environment.GetEnvironmentVariable("APPSETTINGS_ROOT");
+                    if (!string.IsNullOrEmpty(root))
+                    {
+                        var env = builderContext.HostingEnvironment;
+                        config.Sources.Clear();
```

Comment:
What's the `Clear();` for?

Reply:
If you use the docker image unaltered it will already include appsettings at a place listed by default in `config.Sources`. So you either build the image yourself and make sure to remove those files before building, or you have this bit that clears the current settings before applying the new ones.

Reply:
I think a better alternative is to use `SetBasePath`. You don’t need to delete the old files and you can use your secret volume’s settings files.

Reply:
I agree. Will have another look and get back to you.